src/prpy/planning/retimer.py

```diff
@@ -154,6 +154,15 @@ def __init__(self, do_blend=True, do_shortcut=True, blend_radius=0.5,
             'time_limit': float(timelimit),
         })
 
+    @PlanningMethod
+    def RetimeTrajectory(self, robot, path, options=None, **kw_args):
+        from copy import deepcopy
+        if options is None:
+            options = {}
+        new_options = deepcopy(options)
+        new_options['time_limit'] = kw_args.get('timelimit', 3.)
+        return super(HauserParabolicSmoother, self).RetimeTrajectory(
+            robot, path, options=new_options, **kw_args)
```

Comment:
Hauser smoother expects a parameter named 'time_limit'.  Our prpy planning convention is to use 'timelimit'.